src/eradius_server.erl

```diff
@@ -325,6 +326,12 @@ apply_handler_mod(HandlerMod, HandlerArg, Request, NasProp) ->
             {reply, EncReply,{Request#radius_request.cmd, ReplyCmd}};
         noreply ->
             {discard, handler_returned_noreply};
+        {error, timeout} ->
+            S = {NasProp#nas_prop.nas_ip, NasProp#nas_prop.nas_port, Request#radius_request.reqid},
+            NAS = eradius_lib:get_attr(Request, ?NAS_Identifier),
+            NAS_IP = inet_parse:ntoa(NasProp#nas_prop.nas_ip),
+            lager:error(eradius_log:collect_meta(S, Request), "timeout after waiting for response from RADIUS NAS: ~s ~s", [NAS, NAS_IP]),
```

Comment:
Can we also add information about what we were waiting for?
example: timeout waiting for response [...] to Access-Request(1) [...]
The "Access-Request(1)" is quite relevant for operations. (what kind of request / request id)